docs/dev-container.md

```
@@ -39,12 +38,12 @@ For now, you must _rebuild_ the container using `⌘` `⇧` `P`
-    3. Wait for the container to build and open the project in a new window.
+   1. Open the command palette (Cmd+Shift+P on macOS, Ctrl+Shift+P on Windows/Linux) and select **Remote-Containers: Open Folder in Container...**.
+   2. Select the project folder.
+   3. Wait for the container to build and open the project in a new window.
 
 3. :tada: **You're ready to go!**
 
 ## What's Next?
 
-* Please report any issues you encounter with the dev container in [GitHub Issues, with the `devcontainers` label](https://github.com/bedtime-coders/bedstack/labels/devcontainers).
\ No newline at end of file
+* Please report any issues you encounter with the dev container in [GitHub Issues, with the `devcontainers` label](https://github.com/bedtime-coders/bedstack/labels/devcontainers).
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)